authorization-service/src/handlers/basicAuthorizer/handler.ts

```diff
@@ -0,0 +1,17 @@
+import { APIGatewayTokenAuthorizerEvent } from "aws-lambda";
+import { ENCODING_BASE_64, UNICODE_UTF_8, resource, separators} from "../../constants";
+import { Effect } from "../../types/effect";
+import { generatePolicy } from "../../utils/generatePolicy";
+
+export const basicAuthorizer = async (event: APIGatewayTokenAuthorizerEvent) => { 
+    const token = event.authorizationToken.split(separators[0])[1];  
```

Comment:
We should handle cases when the token would not be the expected format

Authorizer should not fall, it should return Deny Policy

What if `event.authorizationToken.split(separators[0])` would return array of 1 element?
And you are trying to get second element in array?

`event.authorizationToken.split(separators[0])[1]`

It would fail

Reply:
done